examples/hosts/app-integration/schema-upgrade/src/migrationInterfaces/migrationTool.ts

```diff
@@ -34,6 +35,11 @@ export interface IMigrationTool extends IEventProvider<IMigrationToolEvents> {
      */
     finalizeMigration(id: string): Promise<void>;
 
+    /**
+     * TaskManager DDS to facilitate the migration execution.
+     */
+    readonly taskManager: ITaskManager;
```

Comment:
This interface generally doesn't reveal the DDSs that can be used for migration, but instead provides special-purpose API surface for migration while keeping them concealed.  Maybe we want to do that here too (I have not thought through in any detail though)?